src/NzbDrone.Core.Test/ParserTests/ParserFixture.cs

```diff
@@ -23,7 +23,9 @@ public class ParserFixture : CoreTest
          * Superman.-.The.Man.of.Steel.1994-06.34.hybrid.DreamGirl-Novus-HD
          * Superman.-.The.Man.of.Steel.1994-05.33.hybrid.DreamGirl-Novus-HD
          * Constantine S1-E1-WEB-DL-1080p-NZBgeek
-         * [TestCase("Valana la Movie FRENCH BluRay 720p 2016 kjhlj", "Valana la Movie")]  Removed 2021-12-19 as this / the regex for this was breaking all movies w/ french in title
+         * Valana la Movie FRENCH BluRay 720p 2016 kjhlj
+         * Valana la Movie TRUEFRENCH BluRay 720p 2016 kjhlj
```

Comment:
TRUEFRENCH test case cannot be removed; This is a valid release in the wild.

Reply:
Okay, we could keep the old regex for TRUEFRENCH and additionally use my new regex for FRENCH and GERMAN. Or I try to change the new regex to match also the TRUEFRENCH release above, but this would make it more complicated.
Do you have some more example for releases in this format?

Reply:
I missed your screenshots. The releases above should not match with my regex, because it's very strict. Therefore, when the language comes after the year, it will be matched be the [standard regex](https://github.com/Radarr/Radarr/pull/6999/files#diff-66eb7a94e4c0bcb08b7dd279734bda078d52ea70a99c0df3dcfaf13e86e0602fL27).

Tomorrow I will test the releases above to make sure.

Reply:
The problem with the TRUEFRENCH test above is, that the year is in the wrong position.

Reply:
from glancing through the ~600 results in movie cats that Prowlarr returned; it seems you're correct - does not appear there are instances with TRUEFRENCH before the name in the wild?

Reply:
Here are the Tests I used to update the regex: https://regex101.com/r/8uEUSr/2
As you can see, releases with a year in the title are not matched with this regex, they are then handled by the second regex (except titles where the Language tag comes before the year `Der.Film.deines.Lebens.German.2011.PAL.DVDR-ETM`)

I tested all the releases in the screenshots above, none of them match, which is the expected result.